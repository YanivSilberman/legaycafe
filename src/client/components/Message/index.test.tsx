import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import Component from './index';

describe('Message component test',() => {
  const renderer = ReactShallowRenderer.createRenderer();

  it('component loads', () => {
    const msg = {
      _id: "",
      text: "",
      user: "",
      createdAt: ""
    };

    expect(renderer.render(
      <Component
        index={1}
        userId="123"
        lastUser="345"
        avatar="123"
        classes={{}}
        {...msg}
      />
    )).toMatchSnapshot()
  })

});
