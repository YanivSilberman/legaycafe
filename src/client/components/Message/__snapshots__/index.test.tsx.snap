// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component test component loads 1`] = `
<Message
  _id=""
  avatar="123"
  classes={
    Object {
      "avatar": "Message-avatar-3",
      "avatarPlaceholder": "Message-avatarPlaceholder-4",
      "message": "Message-message-1",
      "ownMessage": "Message-ownMessage-2",
    }
  }
  createdAt=""
  index={1}
  lastUser="345"
  text=""
  user=""
  userId="123"
/>
`;
